packages/ui_v2/src/components/commonV2/FormField/index.tsx

```diff
@@ -222,6 +223,7 @@ const DurationFieldView = observer(
             onChange={onChange}
             type={"number"}
             min={0}
+            max={99}
```

Comment:
Why is 99 the max? What if I want proposal duration to be longer than 100 days?

Reply:
so we shouldn't define a max then?

Reply:
I don't think so